### PR TITLE
Added a basic readme file

### PR DESCRIPTION
**Pull request by linaori**

### CODE REVIEW

**Comment by yannickl88** [comment]
nice :+1:

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+## Connecting a repository
+
+In order to connect a repository with blog articles, you have to link it via `app:add-blog`. You also have to add a
+commit hook in your repository with the secret provided. This makes sure that both the initial creation and additional
+updates in the repository will sync it to the app.
+
+
+### Repository Layout
+
+The main file you need is the `blog.yml`, this contains the settings of your blog. An example below:
```
Comment (yannickl88) [inline readme.md:10]: You originally used blog.y**a**ml, but I think I prefer blog.yml (without the a). Maybe we should support both (or at least for migration)

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ - `introduction`: links to your intro which will be displayed on the right side next to posts.
+ - `drafts`: which indicates your draft blog posts directory.
+ - `published`: which indicates your published blog posts directory.
+
+
+### Blog Post Statuses
+
+There are three different statuses a blog post can have:
+ - Draft: will be displayed when the url is accessed directly but not listed.
+ - Unpublished: same as draft but in the published directory and will be released if the publish data is passed.
```
Comment (yannickl88) [inline readme.md]: Maybe:
same as draft but in the published directory and have a date in the future. Once the date has been passed they will become published

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+ - Published: when the publish date has been passed in the published directory, also adds Disqus features.
```
Comment (yannickl88) [inline readme.md]: unpublished also have Disqus, only drafts do not.

File: readme.md (hunk 1 above)
Comment (linaori) [inline readme.md:10]: Something about `referrer` and `referer` :(

It would be a good idea to support both yes.

**Pull request merged**